### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by JulieADCL** [ruthlennonatu/bsc23grassroots#22]
As a customer, I want a grant application system that is accessible and inclusive, ensuring that I can use it regardless of my physical abilities

**Comment by l00145352** [comment]
ill see what i can do on this here so i will in what context though

**Comment by L00163597** [comment]
Found some resources:


- [All in One Accessibility](https://pypi.org/project/django-all-in-one-accessibility/) supposedly helps comply with EU regulations for accessibly. 
- [Lighthouse](https://www.foo.software/docs/lighthouse-check-github-action/examples) Googles performance and accessibility testing. Will provide scores that we can customise to fail if too low. Image below as example.

![image](https://github.com/ruthlennonatu/bsc23grassroots/assets/147529915/464a2cff-e4be-479b-ac0d-e3951d93826b)
